### DISCUSSION TITLE
Practice mode?

### DISCUSSION THREAD

**Discussion by fuyuka-nix** [ppy/osu#24580] [Ideas] [answered]
I think that a "practice mode" menu would be a quality life change, a menu like Synthesia bucle menu or the ADOFAI practice mode makes easier to practice that VERY dificult parts that some maps have at the end of the song(or even the whole song).
In my opinion this could me added as a mod or as an option on the settings menu :>

PS:sorry for my grammar, my english level is A2-B1.

**Comment by Rekunan** [comment, marked as answer]
Has been suggested multiple times before, main thread being #13013

**Reply by fuyuka-nix** [reply]
> Has been suggested multiple times before, main thread being #13013

My apologies, next time I'll make sure there isn't another thread like the one I write...
